lib/edit_name.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'nameable'
+
+class EditName
+  include Nameable
+  attr_accessor :nameable
```

Comment:
- [ ] It would be great if after making the Nameable a class you should make this EditName Class a child class of Nameable class to follow the project requirements and decorator pattern 👍